src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/ApplicationInsights/ApplicationInsightsLogger.cs

```diff
@@ -205,7 +223,7 @@ private static void ApplyHttpRequestProperties(RequestTelemetry requestTelemetry
                 return;
             }
 
-            requestTelemetry.Url = request.RequestUri;
+            requestTelemetry.Url = new Uri(request.RequestUri.GetLeftPart(UriPartial.Path));
```

Comment:
I commented on `.Name` to be path. `.Url` is supposed to be a full url or any other data arguments for the request call (if not http).